ohmyk8s.yaml

```diff
@@ -141,13 +141,20 @@
     community.kubernetes.helm_plugin:
       plugin_path: https://github.com/jkroepke/helm-secrets
       state: present
-
 - name: skaffold installer
   hosts: k8sdev, k8sdevlite
   tasks:
+  - name: Determine the correct package link based on architecture
+    set_fact:
+      package_link: >-
+        {%- if ansible_architecture == 'aarch64' -%}
```

Comment:
Does this get evaluated on the controller or target machine?

Reply:
It's on target, i ran the script from amd64 machine controller against aarch64 target machine :

![target_arch](https://github.com/user-attachments/assets/83785cf1-86f0-4d06-8953-3f932892e9e9)

Used task:

```yaml
- hosts: all
  tasks:
    - name: Gather architecture information
      setup:

    - name: Display target machine architecture
      debug:
        msg: "Target machine architecture: {{ ansible_architecture }}"